src/main/java/org/zalando/nakadi/repository/db/TimelineDbRepository.java

```diff
@@ -38,6 +38,10 @@ public TimelineDbRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper
                 eventType);
     }
 
+    public List<Timeline> listTimelines() {
+        return jdbcTemplate.query(BASE_TIMELINE_QUERY + " order by t.tl_order", timelineRowMapper);
+    }
```

Comment:
It's not a good idea to select all timelines (more than 2000 records) only to check if any storage is used. Maybe it's better to create method `isStorageUsed(Storage)`